core/domain/prod_validation_jobs_one_off.py

```diff
@@ -425,6 +438,16 @@ def _validate_commit_cmds_schema(cls, item):
             item: ndb.Model. Entity to validate.
         """
         change_domain_object = cls._get_change_domain_class(item)
+        if change_domain_object is None:
+            # This is for cases where id of entity is invalid
+            # and no commtit command domain object is found for entity.
```

Comment:
Spelling: commit not commtit

entity --> the entity

Reply:
Done.